src/Web/Sfa.Das.Sas.Shared.Components.Web/Views/Shared/_Layout.cshtml

```diff
@@ -56,14 +56,20 @@
     <div class="govuk-width-container">
 
         <div class="govuk-phase-banner">
+            <ul style="float:right">
```

Comment:
Should we have inline styles? maybe one for @jamesmking to move into the das-frontend project?

Reply:
Not on our sites no. But as this is our test app I didn't see an issue as we have no styling currently for a lot of the fat components. Am happy to take the inline styling out and let it sit somewhere else. We've got a story in the backlog for "Polishing up" up this test website. We should discuss with @jamesmking how best to approach the styles especially as at the moment we don't have a anything in the backlog for making a GDS version.

Reply:
Spoke to @jamesmking. He's going to schedule in some time to add some styling for the GDS version.